libcontainer/configs/cgroup_linux.go

```diff
@@ -41,6 +41,13 @@ type Cgroup struct {
 
 	// Rootless tells if rootless cgroups should be used.
 	Rootless bool
+
+	// The host UID that should own the cgroup, or nil to accept
+	// the default ownership.  This should only be set when the
+	// cgroupfs is to be mounted read/write.
+	// Not all cgroup manager implementations support changing
+	// the ownership.
+	OwnerUID *int `json:"owner_uid,omitempty"`
```

Comment:
Is there a need for group as well? crun sets both.

Reply:
I'm not sure.  I don't need it for my use case.  It would be best to discuss it in the OCI runtime-spec PR (https://github.com/opencontainers/runtime-spec/pull/1123) where we are formalising a semantics for container cgroup ownership.